Apps/Admin/Tests/Services/BetaFeatureServiceTests.cs

```diff
@@ -0,0 +1,346 @@
+        /// <returns>
+        /// A <see cref="Task"/> representing the asynchronous unit test.
+        /// </returns>
+        [InlineData(true)]
+        [InlineData(false)]
+        [Theory]
+        public async Task ShouldGetUserAccess(bool profileExists)
+        {
+            // Arrange
+            IEnumerable<Common.Constants.BetaFeature> expected =
+            [
+                Common.Constants.BetaFeature.Salesforce,
+            ];
+
+            ICollection<BetaFeatureCode> betaFeatureCodes =
+            [
+                GenerateBetaFeatureCode(BetaFeature.Salesforce),
+            ];
+
+            IList<UserProfile> userProfiles = profileExists
+                ?
+                [
+                    GenerateUserProfile(Hdid1Email1, Email1, betaFeatureCodes),
+                    GenerateUserProfile(Hdid2Email1, Email1, betaFeatureCodes),
+                ]
+                : [];
+
+            Mock<IUserProfileDelegate> profileDelegateMock = new();
+            profileDelegateMock.Setup(s => s.GetUserProfilesAsync(Email1, true, It.IsAny<CancellationToken>())).ReturnsAsync(userProfiles);
+            IBetaFeatureService service = GetBetaFeatureService(profileDelegateMock: profileDelegateMock);
+
+            if (profileExists)
+            {
+                // Act
+                IEnumerable<Common.Constants.BetaFeature> enumerable = await service.GetUserAccessAsync(Email1);
+                IList<Common.Constants.BetaFeature> actual = enumerable.ToList();
+
+                // Assert
+                Assert.Single(actual);
+                Assert.Equal(expected, actual);
+            }
+            else
+            {
+                // Act and Assert
+                Exception exception = await Assert.ThrowsAsync<NotFoundException>(
+                    async () => { await service.GetUserAccessAsync(Email1); });
+                Assert.Equal(ErrorMessages.UserProfileNotFound, exception.Message);
+            }
+        }
+
+        /// <summary>
+        /// GetBetaFeatureAccessAsync.
+        /// </summary>
+        /// <returns>
+        /// A <see cref="Task"/> representing the asynchronous unit test.
+        /// </returns>
+        [Fact]
+        public async Task ShouldGetBetaFeatureAccess()
+        {
+            // Arrange
+            ICollection<BetaFeatureCode> betaFeatureCodes =
+            [
+                GenerateBetaFeatureCode(BetaFeature.Salesforce),
+            ];
+
+            IList<UserProfile> userProfiles =
+            [
+                GenerateUserProfile(Hdid1Email1, Email1, betaFeatureCodes),
+                GenerateUserProfile(Hdid2Email1, Email1, betaFeatureCodes),
+            ];
+
+            IList<BetaFeatureAccess> betaFeatureAssociations =
+            [
+                GenerateBetaFeatureAccess(Hdid1Email1, Email1),
+                GenerateBetaFeatureAccess(Hdid2Email1, Email1),
+                GenerateBetaFeatureAccess(Hdid3Email2, Email2),
+            ];
```

Comment:
`userProfiles` and `GetUserProfilesAsync()` aren't actually used in the tested method.  The collection is missing a reference for Email2, which would be needed if it were being used.

As a result, the assertion that compares `userProfiles.Count` with `actual.Count` is misleading - yes, there should be 2 results, but that's based on betaFeatureAssociations containing 2 unique email addresses.

I suggest removing `userProfiles`.